Makefile

```diff
@@ -47,7 +47,7 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= controller:v0.13.0
```

Comment:
That should be the real image path from https://quay.io/organization/change-metrics. I'll build on add the image there.